### ISSUE TITLE
I got an error while building project after using the gradle

### ISSUE THREAD

**Issue by alirezaaa** [erizet/SignalA#61]
HI,

I'm getting following error while building my project after adding your dependency. How can I fix it?

```
AAPT err(609058982): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.github.developerpaul123.filepickerlibrary\materialfabmenu\1.4\res\drawable-xhdpi-v4\fab_bg_mini.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(164632686): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.marshalchen.ultimaterecyclerview\library\0.3.17\res\drawable-xhdpi\urv_floating_action_button_fab_bg_mini.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(1284552): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.nhaarman.supertooltips\library\3.0.0\res\drawable-hdpi\quickcontact_drop_shadow.9.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(1528608562): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.github.developerpaul123.filepickerlibrary\materialfabmenu\1.4\res\drawable-hdpi-v4\fab_bg_mini.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(2039405622): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.nhaarman.supertooltips\library\3.0.0\res\drawable-hdpi\tooltip_arrow_down.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(209749332): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.github.developerpaul123.filepickerlibrary\materialfabmenu\1.4\res\drawable-xhdpi-v4\fab_bg_normal.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(631790319): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\multi_image_selector\res\drawable-xxhdpi\asy.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(735118319): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\in.srain.cube\ultra-ptr\1.0.10\res\drawable-xhdpi-v4\ptr_rotate_arrow.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(1528668409): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.github.developerpaul123.filepickerlibrary\materialfabmenu\1.4\res\drawable-mdpi-v4\fab_bg_normal.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(655903408): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.github.developerpaul123.filepickerlibrary\materialfabmenu\1.4\res\drawable-hdpi-v4\fab_bg_normal.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(1383331915): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.github.developerpaul123.filepickerlibrary\materialfabmenu\1.4\res\drawable-mdpi-v4\fab_bg_mini.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(735118319): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.marshalchen.ultimaterecyclerview\library\0.3.17\res\drawable-xhdpi\urv_floating_action_button_fab_bg_normal.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(981662654): C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\exploded-aar\com.nhaarman.supertooltips\library\3.0.0\res\drawable-hdpi\tooltip_arrow_up.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
:app:processDebugManifest
:app:processDebugResources
C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\res\debug\drawable-hdpi-v4\ic_launcher.png: error: Duplicate file.
C:\Users\aessh\AndroidStudioProjects\C-Up\app\build\intermediates\res\debug\drawable-hdpi\ic_launcher.png: Original is here. The version qualifier may be implied.
Error:Execution failed for task ':app:processDebugResources'.
> com.android.ide.common.process.ProcessException: org.gradle.process.internal.ExecException: Process 'command 'C:\Users\aessh\AppData\Local\Android\sdk\build-tools\23.0.1\aapt.exe'' finished with non-zero exit value 1
```

**Comment by keyoh** [comment]
This issue was fixed in pull request #59.

Until a new release is made, you can build the project yourself.

**Comment by alirezaaa** [comment]
@keyoh Thanks but I successfully built that. I removed one of my project's dependencies which has `ic_launcher` images.
